multiview/embed/test.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+
```

Comment:
What is this file for?

Reply:
those were just testing things. i removed them now